sync_crawler/crawler/cts_crawler.py

```diff
@@ -0,0 +1,42 @@
+from proto.news_pb2 import News
+from sync_crawler.crawler.base_crawler import BaseCrawler
+
+
+class CtsCrawler(BaseCrawler):
+
+    def __init__(self, custom_property, *args, **kwargs):
+        # 呼叫父類別的 __init__ 方法，並傳遞必要的參數
+        super().__init__(*args, **kwargs)
```

Comment:
Remove this bacause python will automatically call `super().__init__()` for you.